src/sentry/utils/assets.py

```diff
@@ -1,5 +1,22 @@
 from django.conf import settings
 
+from manifest_loader.utils import _get_manifest, _load_from_manifest
+
+
+def get_manfifest_url(module, key):
+    """
+    Returns an asset URL that is produced by webpack. Uses webpack's manifest to map
+    `key` to the asset produced by webpack. Required if using file contents based hashing for filenames.
+
+    Example:
+      {% manifest_asset_url 'sentry' 'sentry.css' %}
+      =>  "/_static/sentry/dist/sentry.filehash123.css"
+    """
+    manifest_obj = _get_manifest()
```

Comment:
I know this is WIP, but for expensive (presumably this reads the webpack manifest file) function calls like this, you should wrap it with an unbounded function call cache: `@functools.lru_cache(maxsize=None)`. Unbounded is good here because we know it'll only ever be 1 cached call (no arguments), and it doesn't need to evict so it's faster than maxsize=N (there's a fast path for this in CPython).

Reply:
@joshuarli so the manifest library seems to cache this: https://github.com/shonin/django-manifest-loader/blob/dev/manifest_loader/utils.py#L62

Reply:
Oh, that's good.